CountItems/CountWorker.js

```diff
@@ -30,6 +30,11 @@ class CountWorker {
         if (!this.client.client) {
             return callback(new Error('NotConnected'));
         }
+        // 'fromObj' expects that the website configuration is an instance of
+        // WebsiteConfiguration As it it not used in CountItems, we nullify it.
```

Comment:
why would be not be a `WebsiteConfiguration` ? There is probably another root cause, would be best to fix it instead of adding a band-aid...

Reply:
It's not ban-aid: the function we use, `fromObj`, loads an object and expects the website configuration to be of the right instance, it is not the `deSerialize` method, where we accept a non-instanciated JSON object. The latter could be used, but we would need to serialize the object metadata object. I chose to keep the same function, but adapt the object to pass the class `assert`s...

> why would be not be a WebsiteConfiguration ?

Because this object comes from MongoDB, and has no specific class inside, it's just an object with fields.

Reply:
I will alsodefinitely not update arsenal for the count-items job, that we really need to drop today...